docs/reference/ml/df-analytics/apis/index.asciidoc

```diff
@@ -3,6 +3,7 @@ include::ml-df-analytics-apis.asciidoc[leveloffset=+1]
 include::put-dfanalytics.asciidoc[leveloffset=+2]
 include::put-trained-models-aliases.asciidoc[leveloffset=+2]
 include::put-trained-models.asciidoc[leveloffset=+2]
+include::put-trained-model-definition-part.asciidoc[leveloffset=+2]
```

Comment:
> include::put-trained-model-definition-part.asciidoc[leveloffset=+2]

A link will also need to be added in https://github.com/elastic/elasticsearch/blob/master/docs/reference/ml/df-analytics/apis/ml-df-analytics-apis.asciidoc